src/typecheck/mod.rs

```diff
@@ -1259,7 +1396,7 @@ pub fn unify_(
 
 /// Try to unify two row types. Return an [`RowUnifError::IllformedRow`] error if one of the given
 /// type is not a row type.
-pub fn unify_rows(
+pub fn unifyrows(
```

Comment:
Did you do `s/unify_/unify/g`? :smile:

Reply:
Ah, guilty as charged. Well spotted :smiley: